internal/api/external.go

```diff
@@ -104,6 +122,7 @@ func (a *API) ExternalProviderRedirect(w http.ResponseWriter, r *http.Request) e
 	default:
 		authURL = p.AuthCodeURL(tokenString, authUrlParams...)
 	}
+	query.Del("code_challenge")
```

Comment:
nit: move this up to be deleted with "scopes" and "provider" (around lines 103-104)